### PR TITLE
fix alt-text code sample

### PR DESCRIPTION
**Pull request by katharinafluch**
addresses #227 
documentation of the "alt-text" rule was not up to date

### CODE REVIEW

File: docs/rules/alt-text.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ This rule takes one optional object argument of type object:
 ```json
 {
     "rules": {
-        "jsx-a11y/img-has-alt": [ 2, {
+        "jsx-a11y/alt-text": [
+         2, {
```
Comment (ljharb) [inline docs/rules/alt-text.md]: please revert this change

File: docs/rules/alt-text.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ An `<img>` must have the `alt` prop set with meaningful text or as an empty stri
 
 For images that are being used as icons for a button or control, the `alt` prop should be set to an empty string (`alt=""`).
 
-```js
+```html
```
Comment (ljharb) [inline docs/rules/alt-text.md]: this should actually be `jsx`, not `html`.

File: docs/rules/alt-text.md (hunk 1 above)
Comment (katharinafluch) [reply docs/rules/alt-text.md]: In my version (6.0.3) I fought with img-has-alt for a very long time, because I always got 94 errors with this message that this rule is not defined. With alt-text the whole thing works. 

error_img-has-alt

Comment (ljharb) [reply docs/rules/alt-text.md]: ohhh sorry I missed the rule name change. That's a good change :-) I'm talking about the formatting - can you revert back to the original formatting, but keep your rule name change?
